etc/ci/buildbot_steps.yml

```diff
@@ -27,6 +27,7 @@ mac-rel-wpt4:
   - ./mach build --release
   - ./mach test-wpt --release --processes 4 --total-chunks 4 --this-chunk 4 --log-raw test-wpt.log --log-errorsummary wpt-errorsummary.log --always-succeed
   - ./mach filter-intermittents wpt-errorsummary.log --log-intermittents intermittents.log --log-filteredsummary filtered-wpt-errorsummary.log --use-tracker
+  - ./mach test-wpt --release --binary-arg=multiprocess --processes=8 --pref dom.servoparser.async_html_tokenizer.enabled --log-raw test-wpt.log --log-errorsummary wpt-errorsummary.log domparsing html/syntax html/dom/documents html/dom/dynamic-markup-insertion
```

Comment:
A couple points:
* don't use `--binary-arg=multiprocess`, since that makes the tests use content processes.
* `--pref` as an argument to test-wpt does not work, I'm pretty sure. We either need to make it work (which would be very useful), or pass it to the underlying servo binary via `--binary-arg`.